cmd/cli/serve/serve.go

```diff
@@ -267,39 +270,44 @@ func serve(cmd *cobra.Command) error {
 		AuthConfig:            authConfig,
 		IsComputeNode:         isComputeNode,
 		IsRequesterNode:       isRequesterNode,
+		RequesterSelfSign:     config.GetRequesterSelfSign(),
 		Labels:                config.GetStringMapString(types.NodeLabels),
 		AllowListedLocalPaths: allowedListLocalPaths,
 		NodeInfoStoreTTL:      nodeInfoStoreTTL,
 		NetworkConfig:         networkConfig,
 	}
-
 	if isRequesterNode {
+		var cert string
+		var key string
+		var err error
```

Comment:
Don't think you need to delcare these separately, just use `cert, key, err := ...`